extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -791,48 +723,39 @@ public static boolean isHardcodedYoutubeMusicKeyValid() throws IOException,
         // @formatter:on
 
         final var headers = new HashMap<>(getOriginReferrerHeaders(YOUTUBE_MUSIC_URL));
-        headers.putAll(getClientHeaders(HARDCODED_YOUTUBE_MUSIC_KEY[1],
-                HARDCODED_YOUTUBE_MUSIC_KEY[2]));
+        headers.putAll(getClientHeaders("67", HARDCODED_YOUTUBE_MUSIC_CLIENT_VERSION));
```

Comment:
Why did you hardcode "67" here? Wouldn't it be better to have a constant named `HARDCODED_YOUTUBE_MUSIC_CLIENT_NAME`?

Also, are you sure this will never change? (I guess the answer is obviously yes, considering it's the client identifier)

Reply:
We do the same thing for the WEB client, so I thought it was OK.